client/store/cart.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable no-case-declarations */
+/* eslint-disable default-case */
+import axios from 'axios';
+
+/**
+ * ACTION TYPES
+ */
+const SET_CART = 'SET_CART';
+const SET_REMOVE_CART = 'SET_REMOVE_CART';
+const SET_ADD_TO_CART = 'SET_ADD_TO_CART';
+const SET_SUBTRACT_FROM_CART = 'SET_SUBTRACT_FROM_CART';
+/**
+ * INITIAL STATE
+ */
+const initialState = {
+	cartItems: []
+};
+
+/**
+ * ACTION CREATORS
+ */
+// const findOrCreateCart = cart => ({type: FIND_OR_CREATE_CART, cart});
+const setCart = cart => ({type: SET_CART, cart});
+const setAddToCart = product => ({type: SET_ADD_TO_CART, product});
+const setRemoveCart = () => ({type: SET_REMOVE_CART});
+const setSubtractFromCart = product => ({type: SET_SUBTRACT_FROM_CART, product});
+/**
+ * THUNK CREATORS
+ */
+//consider renaming findOrCreate cart to GetCart
+export const findOrCreateCart = userId => async dispatch => {
+	try {
+		const res = await axios.get(`/api/cartItems`); // waiting for routes
+        console.log(res.data)
```

Comment:
Remove `console.log()`

Also remove commented out code below (line 41) if not needed